lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -102,7 +102,7 @@
  * displays the selected value using letters to indicate magnitude (e.g.: 1.5K, 3M, 12B, etc..).
  *
  * <p>With the default style {@link
- * com.google.android.material.R.style#Widget_MaterialComponents_Slider}, colorPrimary and
+ * com.google.android.material.R.style#Widget.MaterialComponents.Slider}, colorPrimary and
```

Comment:
this should be `R.style#Widget_MaterialComponents_Slider` since that's how it would be referenced programmatically

Reply:
I thought this had to be a valid Javadoc link, sorry 😅